src/mixins/object_geometry.mixin.ts

```diff
@@ -602,6 +602,10 @@ export class ObjectGeometry extends ObjectOrigin {
     return this.scale(value / this.height / boundingRectFactor);
   }
 
+  getRetinaScaling() {
+    return this.canvas?.getRetinaScaling() || 1;
```

Comment:
exposed this instead of having to check if canvas exists
I wonder if we should change this to
`return this.canvas?.getRetinaScaling() || config.devicePixelRatio;`

Should the method be protected?